src/components/pages/PointSystem/ProgressCard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="progress-circle">
+    <svg :width="svgSize" :height="svgSize">
+      <circle class="progress-circle__background" :cx="center" :cy="center" :r="radius" :stroke-width="strokeWidth" />
+      <circle
+        class="progress-circle__bar"
+        :cx="center"
+        :cy="center"
+        :r="radius"
+        :stroke-width="strokeWidth"
+        :stroke-dasharray="circumference"
+        :stroke-dashoffset="progressDashOffset"
+        stroke-linecap="round"
+        :transform="'rotate(-90 ' + center + ' ' + center + ')'"
+      />
+    </svg>
+    <token-logo
+      v-if="isTokenImage"
+      class="progress-circle__image"
+      :token="this.getImageSrc(imageName)"
+      :width="imageSize"
+      :height="imageSize"
+    />
+    <img
+      v-else
+      class="progress-circle__image"
+      :src="this.getImageSrc(imageName)"
+      :alt="imageName"
+      :width="imageSize"
+      :height="imageSize"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { components } from '@soramitsu/soraneo-wallet-web';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+import { getImageSrc, isTokenImage } from '@/consts/pointSystem';
+
+@Component({
+  components: {
+    TokenLogo: components.TokenLogo,
+  },
+})
+export default class ProgressCard extends Vue {
+  @Prop({ required: true, type: String })
+  readonly imageName!: string;
+
+  @Prop({ required: true, type: Number })
+  readonly progressPercentage!: number;
+
+  get svgSize(): number {
```

Comment:
This value is not changed during lifecycle. Please define it as `data`, or `prop `with default value